src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -0,0 +1,77 @@
+import d3 from 'd3';
+import _ from 'lodash';
+import $ from 'jquery';
+import { VislibVisualizationsChartProvider } from './_chart';
+import { GaugeTypesProvider } from './gauges/gauge_types';
+
+export function GaugeChartProvider(Private) {
+
+  const Chart = Private(VislibVisualizationsChartProvider);
+  const gaugeTypes = Private(GaugeTypesProvider);
+  const defaults = {
```

Comment:
could we remove this config? we already have additional default configs for the individual chart types.